catalog/plugin/catalog-plugin-metacard-validation/pom.xml

```diff
@@ -98,7 +98,7 @@
                                         <limit>
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.74</minimum>
+                                            <minimum>0.63</minimum>
```

Comment:
On the note of `TestCatalog`, the tests in there will need to be updated to account for the CachingFederationStrategy as well? Namely `testValidationChecker`, `testValidationFiltering`, `testValidationUnenforced`, and `testValidationEnforced`.